packages/components/tests/dummy/app/styles/pages/db-dropdown.scss

```diff
@@ -0,0 +1,22 @@
+// DROPDOWN
+
+.dummy-bg-gray {
+  background-color: var(--token-color-palette-neutral-200);
+  padding: 1rem 2rem;
+}
+
+.dummy-dropdown-hide {
+  display: none;
+}
+
+.dummy-dropdown-states-grid {
+  display: grid;
+  grid-gap: 0.5rem;
+  grid-template-columns: 1fr 1fr 1fr 1fr 1fr;
```

Comment:
```suggestion
  grid-template-columns: repeat(5 1fr);
```

Reply:
good idea! I'll update it across all the components pages. thank you